lib/checkPullRequestLabels.js

```diff
@@ -36,7 +35,7 @@ const {
 } = require('../userWhitelist.json');
 const DEFAULT_CHANGELOG_LABEL = 'REVIEWERS: Please add changelog label';
 const PR_LABELS = ['dependencies', 'stale', DEFAULT_CHANGELOG_LABEL];
-
+const utilityModule = require('./utils');
```

Comment:
Can this require statement be added on line 28?
`}, utilityModule = require('./utils');`

Reply:
No, it throws a syntax error `Parsing error: Unexpected token ,eslint`